vector/src/main/res/values/strings.xml

```diff
@@ -3088,4 +3088,10 @@
     <string name="settings_troubleshoot_test_current_endpoint_failed">Cannot find the endpoint.</string>
     <string name="settings_troubleshoot_test_current_gateway_title">Gateway</string>
     <string name="settings_troubleshoot_test_current_gateway">Current gateway: %s</string>
+
+    <!-- Live Location Sharing Labs Flag Promotional BottomSheet -->
+    <string name="live_location_labs_promotion_title">Live location sharing</string>
+    <string name="live_location_labs_promotion_description">Please note: this is a labs feature using a temporary implementation. This means you will not be able to delete your location history, and advanced users will be able to see your location history even after you stop sharing your live location with this room.</string>
```

Comment:
is this the final copy? it's a little bit _techy_~

Reply:
Yes it is. I think it is not so hard to understand for users.